### PR TITLE
Add about us and edit DG

### PR DESCRIPTION
**Pull request by jerome-neo**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Ang Jun Kang
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/junkang.png" width="200px">
```
Comment (venuslimm) [inline docs/AboutUs.md:13]:
```suggestion
<img src="images/Irminrics.png" width="200px">
```

File: docs/AboutUs.md [hunk 2]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
 
-* Role: Project Advisor
+* Role: Team Lead
+* Responsibilites: Pull request approver
 
-### Jane Doe
+### Jerome Neo
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jerome.png" width="200px">
```
Comment (venuslimm) [inline docs/AboutUs.md:23]:
```suggestion
<img src="images/jerome-neo.png" width="200px">
```

File: docs/AboutUs.md [hunk 3]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
-* Role: Team Lead
-* Responsibilities: UI
+* Role: Developer
+* Responsibilities: Documentation, Testing
 
-### Johnny Doe
+### Lim Jia Yi Venus
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/venus.png" width="200px">
```
Comment (venuslimm) [inline docs/AboutUs.md:33]:
```suggestion
<img src="images/venuslimm.png" width="200px">
```

File: docs/AboutUs.md [hunk 4]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: Documentation Quality, Integration 
 
-### Jean Doe
+### Lin Weilin
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/weilin.png" width="200px">
```
Comment (venuslimm) [inline docs/AboutUs.md:43]:
```suggestion
<img src="images/weilin1202.png" width="200px">
```

File: docs/AboutUs.md [hunk 5]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Testing, Code Quality
 
-### James Doe
+### Song Yi, Ina
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/ina.png" width="200px">
```
Comment (venuslimm) [inline docs/AboutUs.md:53]:
```suggestion
<img src="images/inas0ng.png" width="200px">
```

**Review by venuslimm — CHANGES_REQUESTED** [review]
Added suggestions for image names to follow the given convention.

File: docs/AboutUs.md [hunk 6]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
+### Ang Jun Kang
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/junkang.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/Irminrics)]
+[[portfolio](team/junkang.md)]
```
Comment (inas0ng) [inline docs/AboutUs.md:16]:
```suggestion
[[portfolio](team/Irminrics.md)]
```

File: docs/AboutUs.md [hunk 7]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
-### Jane Doe
+### Jerome Neo
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jerome.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/jerome-neo)]
+[[portfolio](team/jerome.md)]
```
Comment (inas0ng) [inline docs/AboutUs.md:26]:
```suggestion
[[portfolio](team/jerome-neo.md)]
```

File: docs/AboutUs.md [hunk 8]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
 
-### Johnny Doe
+### Lim Jia Yi Venus
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/venus.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/venuslimm)]
+[[portfolio](team/venus.md)]
```
Comment (inas0ng) [inline docs/AboutUs.md:36]:
```suggestion
[[portfolio](team/venuslimm.md)]
```

File: docs/AboutUs.md [hunk 9]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
-### Jean Doe
+### Lin Weilin
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/weilin.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/weilin1202)]
+[[portfolio](team/weilin.md)]
```
Comment (inas0ng) [inline docs/AboutUs.md:46]:
```suggestion
[[portfolio](team/weilin1202.md)]
```

File: docs/AboutUs.md [hunk 10]
```diff
@@ -5,55 +5,55 @@ title: About Us
...
-### James Doe
+### Song Yi, Ina
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/ina.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/inas0ng)]
+[[portfolio](team/ina.md)]
```
Comment (inas0ng) [inline docs/AboutUs.md:56]:
```suggestion
[[portfolio](team/inas0ng.md)]
```

**Review by inas0ng — CHANGES_REQUESTED** [review]
Added suggestions for portfolio names to follow the given convention.

**Pull request closed without merging**